src/Raven.Server/Smuggler/Documents/CsvStreamSource.cs

```diff
@@ -153,13 +155,26 @@ public DatabaseRecord GetDatabaseRecord()
 
         public IEnumerable<DocumentItem> GetDocuments(List<string> collectionsToExport, INewDocumentActions actions)
         {
+            var line = 0;
             while (_csvReader.Read())
             {
+                line++;
+
                 if (ProcessFieldsIfNeeded())
                     continue;
 
                 var context = actions.GetContextForNewDocument();
-                yield return ConvertRecordToDocumentItem(context, _csvReader.Context.Record, _csvReaderFieldHeaders, _collection);
+                DocumentItem item = null;
+                try
+                {
+                    item = ConvertRecordToDocumentItem(context, _csvReader.Context.Record, _csvReaderFieldHeaders, _collection);
+                }
+                catch (Exception e)
+                {
+                    _result.AddError($"Fail to parse CSV line {line}, Error:{e.Message}");
```

Comment:
Include the full error message?

Reply:
The call stack won't give the user any additional info and will make huge messages in the studio, give a line number the user can check his input.